paramfetch.go

```diff
@@ -203,11 +203,8 @@ func doFetch(ctx context.Context, out string, info paramFile) error {
 	}
 	defer resp.Body.Close()
 
-	bar := pb.New64(fStat.Size() + resp.ContentLength)
-	bar.Set64(fStat.Size())
-	bar.Units = pb.U_BYTES
-	bar.ShowSpeed = true
-	bar.Start()
+	bar := pb.New64(fStat.Size() + resp.ContentLength).
+		SetCurrent(fStat.Size()).Start()
```

Comment:
What about the unit?

Reply:
Calling `NewProxyReader` does that automatically.